company_linkage/sql/organizations.sql

```diff
@@ -47,7 +59,7 @@ SELECT
         UNNEST(market) m )) AS market )
 FROM (
   SELECT
-    CSET_id,
+    COALESCE(MAX(legacy_cset_id), 4000+new_cset_id) AS CSET_id,
```

Comment:
Then here we're taking the legacy cset id and coalescing it with 4000 + the new cset id to get the final cset id.

Okay, I think I follow!